### DISCUSSION TITLE
[Feature Request]: Resize the menu bar icon

### DISCUSSION THREAD

**Discussion by mchev** [beyondcode/herd-community#375] [Ideas]
### Feature Description

Well, it's not an evolution or a bug, but a small detail that will make a big difference :)

![Herd icon](https://i.imgur.com/PJiszhD.png)

**Comment by coreysmithdesign** [comment]
I'm not myself a huge fan of menu bar icons. I understand why they are liked. I just appreciate being able to turn them off. 

For me, a lot of icons can be an actual menu item, or information and settings in the apps UI. I can figure out the run state and access interfaces using the dock/launchpad/launchbar/alfred/raycast (Herd doesn't seem to respond to these yet though?)

My preference is to use the menu bar for actions I do a lot, from anywhere (change wifi, toggle system settings, select quick functions à la CleanShotX). I don't like to surrender the space for license management, checking for updates, doc shortcuts, subscribing to newsletters, finding on discord, or as a sort of mini-dock app launcher.

If I'm being required to have menu bar icon though, the HELO icon seems right to me. The Herd menu bar icon seems visually heavy, and then increased in size, compared to all the other menu bar icons, as highlighted above.
